test/functional/testlib.bash

```diff
@@ -802,7 +802,31 @@ create_bundle() {
 
 		EOM
 	}
-	
+
+	add_dirs_recursively() {
+
+		if [[ "$val" != "/"* ]]; then
+			val=/"$val"
+		fi
+		# if the directories the file is don't exist, add them to the bundle
+		# there are a few exceptions, the dir of the tracking file /usr/share/clear/bundles,
```

Comment:
I wouldn't call this an exception, since `/usr/share/clear/` is simply subtracted out by mixer due to those dirs being in `os-core`. Hence it looks like an exception because every bundle will have `/usr/share/clear/bundles/bundle` but not any of `/usr/share/clear/bundles/`. I'm not sure if you want to hardcode in the "exception" here.

Reply:
you are right, my choice of words may not be the best here. What I meant is that if we don't consider that file as a "separate case" (not exception), then every test bundle we create will have at least 5 files/dirs (/usr, /usr/share, /usr/share/clear, usr/share/clear/bundles, and  /usr/share/clear/bundles/<bundle_name>), which may be a little too much for most tests. If that file is considered separately from the others, then it is possible to create a bundle with just one file (/usr/share/clear/bundles/<bundle_name>).

Reply:
I will re-word the phrase

Reply:
+1 on keeping the same functionality, but rewording the comment.